PSKoans/Public/Show-Karma.ps1

```diff
@@ -80,6 +85,30 @@ function Show-Karma {
                 $KoanLocation | Invoke-Item
             }
         }
+        'OpenFile' {
+            try {
+                $Results = Get-Karma @GetParams
+            }
+            catch {
+                $PSCmdlet.ThrowTerminatingError($_)
+            }
+            $UnformatedPath     = $Results.Meditation
+            $PSKoansFilePath = $UnformatedPath.Substring($UnformatedPath.IndexOf(', ')+2,$UnformatedPath.IndexOf(': ')-($UnformatedPath.IndexOf(', ')+2))
+            $PSKoansLine = $UnformatedPath.Substring($UnformatedPath.IndexOf(': line ')+7,$UnformatedPath.LastIndexOf(("`n"))-($UnformatedPath.IndexOf(': line ')+7))    
+            $FullArgument = "-g ${PSKoansFilePath}:${PSKoansLine}"
```

Comment:
```suggestion
            $FullArgument = '-g "{0}":{1}' -f $PSKoansFilePath, $PSKoansLine
```

We need to have quotes around the path, in case the path has a space in it. I'd also recommend using simpler variable names; `$LineNumber` and `$FilePath` are sufficient here, I think. 🙂